src/snowflake/cli/_plugins/nativeapp/v2_conversions/compat.py

```diff
@@ -293,15 +293,6 @@ def wrapper(*args, **kwargs):
 
                 # Override the project definition so that the command operates on the new entities
                 cm.override_project_definition = pdfv2
-
-                # Override the template context so that templates refer to the new entities
-                # Reuse the old ctx.env and other top-level keys in the template context
-                # since they don't change between v1 and v2
-                pdfv2_dump = pdfv2.model_dump(
-                    exclude_none=True, warnings=False, by_alias=True
-                )
-                new_ctx = pdfv2_dump | dict(env=cm.template_context["ctx"]["env"])
-                cm.override_template_context = cm.template_context | dict(ctx=new_ctx)
```

Comment:
stop overriding the template context, templates should be able to refer to fields as they exist in the PDF on-disk, since that's the one that the user sees